### PR TITLE
Adding tfctl logs command

### PR DESCRIPTION
**Pull request by luizbafilho**
- adds the `tfctl logs` command. It works pretty much like `flux logs` with a couple of adaptations to tf-controller reality.

### CODE REVIEW

File: cmd/tfctl/logs.go [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+	  # Stream logs for a particular log level
+	  tfctl logs --follow --level=error --all-namespaces
+
+	  # Filter logs by name and namespace
+	  tfctl logs --name=podinfo --namespace=default
+
+	  # Print logs when Flux is installed in a different namespace than flux-system
+	  tfctl logs --controller-namespace=my-namespace
+		`,
```
[the remark this answers is not in the document]
Comment (yitsushi) [reply cmd/tfctl/logs.go:67]: Correct me @chanwit, @yiannistri please, if i'm wrong.

[the remark this answers is not in the document]
Comment (yitsushi) [reply cmd/tfctl/logs.go:67]: Good question, @chanwit ?

[the remark this answers is not in the document]
Comment (chanwit) [reply cmd/tfctl/logs.go:67]: > Runner logs are not easy to access, as the runner pod name is generated.
Runner name is deterministic. For example if the object name is "helloworld", the pod name is "helloworld-tf-runner".

Currently people run `kubectl logs pod/helloworld-tf-runner --follow` and retry when the pod's not found, for example.

[the remark this answers is not in the document]
Comment (chanwit) [reply cmd/tfctl/logs.go:67]: > you'll get a not found from the API. What should we do then?

Just retry.

[the remark this answers is not in the document]
Comment (chanwit) [reply cmd/tfctl/logs.go:67]: I would suggest you try to view runner logs with `kubectl` and replicate your experience to this command.

**Review by yitsushi — APPROVED** [review]

**Pull request merged**